### ISSUE TITLE
waterway+tunnel=culvert should be behind riverbank

### ISSUE THREAD

**Issue by Frankkkkk** [openstreetmap-carto/openstreetmap-carto#2272]
Waterways being tunnels (tunnel=culvert) should be behind  riverbanks, else things like [this](http://www.openstreetmap.org/?mlat=46.19957&mlon=6.12801#map=19/46.19957/6.12801) might happen.

**Comment by pnorman** [comment]
![image](https://cloud.githubusercontent.com/assets/1190866/17465981/dcb824c6-5cb8-11e6-89b7-a5feaeeafe1e.png)

The north-south way passing under the bridge is a waterway=river tunnel=culvert. You can see the dashed casing between the bridge and the forest.

I might wonder about a tagging error, except we have normal waterway=river invisible against water areas.

**Comment by Frankkkkk** [comment]
(Yeah, didn't know where to host the image, sorry; thanks.)

Actually I did tag this river (w432696896) . It exists in real life though I'm not sure that the tags are perfect. I suppose that the problem wouldn't disappear changing them ?

**Comment by HolgerJeromin** [comment]
Just for your information: github hosts the image itself :-)

**Comment by matkoniecz** [comment]
Is http://www.openstreetmap.org/way/432696896#map=19/46.19926/6.12839 mapped correctly?

Is it really a tunnel ending in the middle of the river?

**Comment by Frankkkkk** [comment]
yes

**Comment by Tomasz-W** [comment]
related to https://github.com/gravitystorm/openstreetmap-carto/issues/405 (water layers)

**Comment by jeisenbe** [comment]
I disagree with this suggestion. Waterway tunnels, like road tunnels, should show over all types of landcover, including water areas. 

Otherwise, a canal or aqueduct tunnel which passes under a river or lake, but does not connect to the water area, would appear to be interrupted

**Comment by dch0ph** [comment]
@imagico I came across this in collating waterway issues. I'm pretty sure is a non-issue / wont-fix.

**Comment by imagico** [comment]
Indeed - this is a non-issue now since we changed the layer order of water lines and polygons.  See for example https://openstreetmap.org/way/75255454

But i concur with @jeisenbe that as far as waterway tunnes are concerned this is undesirable.  This could be changed by rendering waterway tunnels in a separate layer like `waterway-bridges`.  Not sure if this is worth the complexity though.

There is the related matter of layering of water polygons - which is covered by #1608 and #3312.
